aiida_vasp/workchains/bands.py

```diff
@@ -208,7 +208,11 @@ def results(self):
         """Attach the remaining output results."""
 
         workchain = self.ctx.workchains[-1]
-        self.out_many(self.exposed_outputs(workchain, self._next_workchain))
+        out_dict = self.exposed_outputs(workchain, self._next_workchain)
+        bands = out_dict.pop('bands', None)
+        self.out_many(out_dict)
+        if bands is not None:
```

Comment:
Since we now introduce a check of the content of the returned output of the calculation, I suggest we also set an exit status here that is specific to the lacking return of the bands data. This can be defined at the top. Just pick a number, say 200x something for now. We need to reshuffle these exit code numbers and I was planning to do this for the coming release.